lib/Game.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+class Game
+  # This class contain all function of the game overflow
+  @debug_on = false
+  def weapon_confimation(weapon_p1, weapon_p2)
+    puts 'test1' if @debug_on
```

Comment:
@bolah2009 thanks for your explanation. But i'm afraid that what you guys are suggesting is not helping me. Also my mentor is doing same thing just telling me things i know (like remove puts ... yeah of i will remove them if i would know how to do that) . Always when i use 'rubocop -a' instead of reducing linter errors it shows me runtime error in my code.  I'm just tired of this a cycle that never ends :(

Reply:
@R4Ajeti, how can I help? What issue are you having currently?